analysis/5/countries_with_highestdeaths_10to29_yearsold.ipynb

```diff
@@ -0,0 +1,316 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "country = sort_total[\"Country\"].head(10)\n",
+    "Total = sort_total[\"total_from_10to29_yearsold\"].head(10)\n",
```

Comment:
These 3 lines of code: 

```
sort_total = suicide_rate_data.sort_values("total_from_10to29_yearsold", ascending = False)
country = sort_total["Country"].head(10)
Total = sort_total["total_from_10to29_yearsold"].head(10)
```

can be streamlined into:
`Total = suicide_rate_data.nlargest(10, 'total_from_10to29_yearsold')`